portfolio/src/main/webapp/index.html

```diff
@@ -14,27 +14,45 @@ <h1>Hannah's Portfolio</h1>
       <button onclick="addQuote()" class="button">Click here to get a random quote!</button>
       <div id="greeting-container"></div>
       <p>Hannah is a big fan of food, like the dessert shown below (which was really good):</p>
-      <img src="/images/food.jpg">
-      <!-- <p>Hannah's major is Symbolic Systems, which ties together</p>
-      <ul id="symsys-container"></ul> -->
+      <img src="/images/food.jpg" width=600px>
+    </div>
+    <div id="comment-container">
       <h3>Feel free to leave comments!</h3>
-      <form action="/data" method="POST">
+      <form action="/data" class="comment-form" method="POST">
         <p>Name:</p>
-        <input name="name">
+        <input name="name" required>
         <p>Comment:</p>
-        <input name="comment">
+        <input name="comment" required>
+        <p>Emoji (optional):</p>
+        <select name="emoji" id="emoji" value="None">
+            <option value="None">None</option>
+            <option value="😄">😄</option>
+            <option value="😍">😍</option>
+            <option value="😴">😴</option>
+            <option value="🤧">🤧</option>
+            <option value="😫">😫</option>
+            <option value="🥺">🥺</option>
+            <option value="🐕">🐕</option>
+            <option value="🦙">🦙</option>
+            <option value="🙊">🙊</option>
+            <option role="img" aria-label="hamburger" value="🍔">🍔</option>
```

Comment:
Is there a reason this one needs to be different?

Reply:
Oh I saw online that there's the possibility that some browsers don't always know how to translate/display the emojis, so the way in line 38 ensures that they do -- so if one of you guys has trouble viewing the emojis when you leave a comment, I can easily copy-paste the lines everywhere